service/history/history_engine2_test.go

```diff
@@ -228,11 +227,19 @@ func (s *engine2Suite) SetupTest() {
 	s.historyEngine = h
 }
 
+func (s *engine2Suite) SetupSubTest() {
+	s.SetupTest()
+}
```

Comment:
This is, unfortunately, necessary to execute the setup before each test _and_ sub test.